tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -23690,7 +23690,7 @@ TEST_F(EnergyPlusFixture, UnitarySystemModel_MultiSpeedFanWSHP_Test)
                      latOut);
     // first speed heating
     EXPECT_NEAR(thisSys.m_SpeedRatio, 0.0, 0.0001);
-    EXPECT_NEAR(thisSys.m_CycRatio, 0.81485929, 0.0001);
+    EXPECT_NEAR(thisSys.m_CycRatio, 0.81485980, 0.0001);
```

Comment:
Verified this unit test calls the modified function and therefore the answer changes slightly.